framework/contracts/account/manager/src/commands.rs

```diff
@@ -902,14 +915,10 @@ pub fn update_suspension_status(
     Ok(response.add_abstract_attributes(vec![("is_suspended", is_suspended.to_string())]))
 }
 
-pub fn update_ibc_status(
+pub fn update_ibc_status_internal(
```

Comment:
Can you add comment why it should only be used internally?

Reply:
And in general the convention is to put on underscore before the name (and not include the "internal" in the name) 

So name would be `_update_ibc_status(...)`